readme.md

```diff
@@ -132,20 +132,20 @@ slugify('foo@unicorn', {
 //=> 'foo-at-unicorn'
 ```
 
-##### customReplacements
+##### preserveLeadingUnderscore
 
 Type: `boolean`\
 Default: `false`
 
-Allow leading underscore as an escape `_foo_bar` → `_foo-bar`.
+It preserves leading underscore: `_foo_bar` → `_foo-bar`.
```

Comment:
I think could have been better written.

```suggestion
If your string starts with an underscore, it will be preserved in the slugified string.
```